.github/workflows/features.yml

```diff
@@ -1,7 +1,4 @@
 on:
-  push:
```

Comment:
This was removed to ensure that CI was not run twice in certain circumstances where PR was opened from a branch on the same repo.